lib/Controller/NavigationController.php

```diff
@@ -167,6 +170,7 @@ public function navigate(string $path = ''): TemplateResponse {
 					if ($cloudAddress !== null) {
 						$this->configService->setCloudUrl($cloudAddress);
 					} else {
+						$this->initialStateService->provideInitialState(Application::APP_NAME, 'serverData', $data['serverData']);
```

Comment:
Could still be simplified a bit since `$data` only has the `serverData` key, so it could just be a one-dimensional array in the code starting from line 147.

Reply:
thanks! I've changed this as well...